src/components/alerting.tsx

```diff
@@ -726,6 +726,13 @@ const AlertsDetailsPage_: React.FC<AlertsDetailsPageProps> = ({ history, match }
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const labels: PrometheusLabels = React.useMemo(() => alert?.labels, [labelsMemoKey]);
 
+  const ruleLabelsMemoKey = JSON.stringify(rule?.labels);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const ruleLabels: PrometheusLabels = React.useMemo(() => rule?.labels, [ruleLabelsMemoKey]);
+
+  const combinedLabels = { ...labels, ...ruleLabels };
+  delete combinedLabels?.prometheus;
```

Comment:
What's the reason for deleting the `prometheus` label? If it should be deleted, let's add a comment explaining  why.

Reply:
This code has been removed in latest commit.